10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,134 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>BILP Indie Rock Playlists</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Pacifico' rel='stylesheet' type='text/css'>
+
+
 </head>
+
 <body>
+    <main>
```

Comment:
great start on this! looking good. for the nav items to line up with the heading words, make sure that the h3's width isn't 100% and the id list is floated right.